src/infrastructure/routes/result.handler.ts

```diff
@@ -69,7 +69,7 @@ function handleFailure(result: Failure): never {
     case ConseillerMiloSansStructure.CODE:
     case JeuneMiloSansStructure.CODE:
     case JeuneMiloSansIdDossier.CODE:
-    case MaxInscritsDepasse.CODE:
+    case NombrePlacesInsuffisant.CODE:
```

Comment:
tu vas l'intérpréter spécifiquement coté front pour afficher un bon msg d'erreur ?

Reply:
Nan, c'est pas censé arrivé parce que le mobile n'affichera le bouton de connexion que si y'a suffisamment de place.

Reply:
attention le mobile utilise bcp le cache... donc il se peut qu'il a un nb de places pas frais
et meme s'il a un nb de places frais... entre le moment ou il clique sur s'inscrire et le moment ou l'api traite la requete il se peut qu'une autre personne a pris la derniere place
donc si on peut demander au jeune Adrien de gérer ce jeune cas d'erreur avec un ptit message explicite it would be nice (et ça nous éparge des cas de support et des recherches dans les logs pour rien)

Reply:
C'est noté